plugins/janus_streaming.c

```diff
@@ -575,6 +587,29 @@ void *janus_streaming_watchdog(void *data) {
 	return NULL;
 }
 
+static int janus_streaming_lookup_network_interface(const struct ifaddrs *ifas, const char *iface, janus_network_address *result) {
+	if(!result) {
+		return -1;
+	} else if(iface) {
+		janus_network_query_config q;
+		if(janus_prepare_network_device_query(iface, janus_ip_network_query_options_ipv4 | janus_ip_network_query_options_name, &q)) {
+			JANUS_LOG(LOG_ERR, "Unable to prepare query of network devices by name and/or IPv4 address for values matching: %s\n", iface);
+			return -1;
+		} else {
+			const struct ifaddrs *found = janus_query_network_devices(ifas, &q);
+			if(!found || janus_get_network_device_address(found, result)) {
+				JANUS_LOG(LOG_INFO, "Unable to retrieve address from matching network device for: %s\n", iface);
```

Comment:
In this case this would result in no address being returned and so a failure to create the mountpoint, so I think LOG_ERR is more appropriate. But nevermind that, it's things I can change later on (also because as I mentioned I plan to implement some helper methods in the ip-utils anyway).